l3kernel/l3basics.dtx

```diff
@@ -1186,9 +1186,9 @@
 %     \cs{cs_if_exist:NTF} \meta{control sequence} \Arg{true code} \Arg{false code}
 %   \end{syntax}
 %   Tests whether the \meta{control sequence} is currently defined
-%   (whether as a function or another control sequence type). Any
-%   definition of \meta{control sequence} other than \tn{relax}
-%   evaluates as \texttt{true}.
+%   (whether as a function or another control sequence type),
+% and its meaning is not the primitive \tn{relax} token
+%(in which it differs from \cs{if_cs_exist:N}).
```

Comment:
Keep indent.
```suggestion
%   (whether as a function or another control sequence type),
%   and its meaning is not the primitive \tn{relax} token
%   (in which it differs from \cs{if_cs_exist:N}).
```

Reply:
Another aspect: doc of `\if_cs_exist:w` mentions `\scan_stop:` instead of `\relax`. Do we want to improve consistency and also use `\scan_stop:` here?